src/Entity/Dataset.php

```diff
@@ -147,7 +147,8 @@ class Dataset extends Entity
      *
      * @var DatasetSubmission
      */
-    #[ORM\OneToOne(targetEntity: 'DatasetSubmission')]
+    #[ORM\OneToOne(targetEntity: 'DatasetSubmission', cascade: ['remove'])]
```

Comment:
I still don't understand why this is a OneToOne here. The relationship between dataset and dataset submission from any perspective was never a one to one.

Reply:
Here it is OneToOne because one submission is the active/approved submission, the rest are kept in datasetSubmissionHistory.